model/internal/data/traceid_test.go

```diff
@@ -41,14 +41,12 @@ func TestTraceIDHexString(t *testing.T) {
 
 func TestTraceIDEqual(t *testing.T) {
 	tid := NewTraceID([16]byte{})
-	assert.True(t, tid.Equal(tid))
 	assert.True(t, tid.Equal(NewTraceID([16]byte{})))
 	assert.False(t, tid.Equal(NewTraceID([16]byte{1})))
 
 	tid = NewTraceID([16]byte{0x12, 0x34, 0x56, 0x78, 0x12, 0x34, 0x56, 0x78, 0x12, 0x34, 0x56, 0x78, 0x12, 0x34, 0x56, 0x78})
-	assert.True(t, tid.Equal(tid))
-	assert.False(t, tid.Equal(NewTraceID([16]byte{})))
```

Comment:
Just confirming was this removed because it was already tested in line 46?

Reply:
assert.True(t, tid.Equal(tid)) the "warning" is about passing the same instance to a instance func.

Reply:
I was referring to the second line

`assert.False(t, tid.Equal(NewTraceID([16]byte{})))`